include/picongpu/fields/incidentField/profiles/Free.def

```diff
@@ -30,7 +30,7 @@ namespace picongpu
         {
             namespace profiles
             {
-                /** Free profile tag using given functors
+                /** Free profile tag usingInvalid number of grid distributions. given functors

```

Comment:
I do not understand that comment/description - but this is fine

Reply:
This should not be part of the PR, I think I was in the wrong window of my IDE and past it in.